### PR TITLE
feat: bump yaru and flutter versions

### PR DESCRIPTION
**Pull request by d-loose**
See https://github.com/canonical/ubuntu-desktop-provision/pull/179

### CODE REVIEW

**Comment by d-loose** [comment]
Resolves #211 thanks to the upstream fix by @Jupi007 :)

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+flutter 3.13.7-stable
```
Comment (tim-hm) [inline .tool-versions:1]: Is there a difference between `3.13.7-stable` and `3.13.7`?

File: pubspec.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ publish_to: "none"
 
 environment:
   sdk: ">=3.0.0 <4.0.0"
-  flutter: ">=3.10.0"
+  flutter: ">=3.13.0"
```
Comment (tim-hm) [inline pubspec.yaml:7]: I tihnk we should fix this to the same version in .tool-versions.

WDYT?

File: snap/snapcraft.yaml [hunk 3]
```diff
@@ -18,14 +18,16 @@ slots:
...
 parts:
   flutter-git:
-    source: https://github.com/flutter/flutter.git
-    source-tag: 3.10.6
-    source-depth: 1
+    source: .
+    override-pull: |
+      craftctl default
+      FLUTTER_VERSION=$(sed -n "s/^flutter \([0-9.]\+\).*/\1/p" .tool-versions)
+      git clone -b $FLUTTER_VERSION --depth 1 https://github.com/flutter/flutter.git
```
Comment (tim-hm) [inline snap/snapcraft.yaml:25]: Nice idea, lets see how this plays out :)

**Review by tim-hm — APPROVED** [review]
A few comments for you to consider, otherwise LGTM

File: .tool-versions (hunk 1 above)
Comment (d-loose) [reply .tool-versions:1]: As discussed during the standup: it's asdf's way of dealing with [flutter channels](https://docs.flutter.dev/release/upgrade#switching-flutter-channels).
```
$ asdf list all flutter | tail
3.13.4-stable
3.15.0-15.1.pre-beta
3.13.5-stable
3.15.0-15.2.pre-beta
3.13.6-stable
3.16.0-0.1.pre-beta
3.13.7-stable
3.16.0-0.2.pre-beta
3.13.8-stable
3.16.0-0.3.pre-beta
```

File: pubspec.yaml (hunk 2 above)
Comment (d-loose) [reply pubspec.yaml]: Fixing this to 3.13.7 then

**Pull request merged**